src/ascii.rs

```diff
@@ -274,7 +274,8 @@ impl SeqVec for Vec<u8> {
     }
 
     fn random(n: usize) -> Self {
-        let mut rng = rand::rng();
-        (0..n).map(|_| rng.random::<u8>()).collect()
+        let mut seq = vec![0; n];
```

Comment:
Guess we could look into maybeuninit here, but whatever, right?

Allocating zeros is cheap, and quite possibly it's optimizing it away anyway.